src/Symfony/Bridge/Twig/Cache/ReadOnlyFilesystemCache.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Twig\Cache;
+
+use Twig\Cache\FilesystemCache;
+
+/**
+ * Implements a cache on the filesystem that can only be read, not written to.
+ *
+ * @author Quentin Devos <quentin@devos.pm>
+ */
+class ReadOnlyFilesystemCache extends FilesystemCache
```

Comment:
Same here (be part of Twig)?

Reply:
Moved to Twig, proposed in twigphp/Twig#4171